PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/platformPlugin.xml

```diff
@@ -6,6 +6,7 @@
     <html>
       <h4>Features:</h4>
       <ul>
+        <li>Compatibility with Rider 2020.3 EAP</li>
         <li>Add Azurite storage emulator integration <a href="https://github.com/JetBrains/azure-tools-for-intellij/issues/312">#312</a></li>
```

Comment:
Maybe, delete other changes (we have already released all of them)? I think it might be more clear to say that we make a release with 203 EAP1 support only to not confuse users about a new version changes.